types/react-leaflet/v1/index.d.ts

```diff
@@ -12,6 +12,11 @@ import * as React from 'react';
 
 export type Children = React.ReactNode | React.ReactNode[];
 
+export interface Viewport {
+    center: [number, number] | null | undefined;
```

Comment:
These don't need `| null`

Reply:
It's typed `?[number, number]` in Flow, and optional type in Flow translate to `| null | undefined` in TypeScript